addons/mail/static/src/chatter/web/form_renderer.js

```diff
@@ -26,6 +26,19 @@ patch(FormRenderer.prototype, {
         }
         this.uiService = useService("ui");
 
+        this.env.bus.addEventListener("add_padding_emoji_picker", ({ detail: { isPadding } }) => {
```

Comment:
Name of the event is not proper. As you are doing both add and remove of the class. It could be `toggle_emoji_picker_padding`. And the `isPadding` parameter could be `addPadding`.